library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -2634,9 +2634,10 @@
     <string name="failed_to_access_secure_storage">Failed to access secure storage</string>
     <string name="bad_passphrase_key_reset_all_action">Forgot or lost all recovery options? Reset everything</string>
     <string name="secure_backup_reset_all">Reset everything</string>
-    <string name="secure_backup_reset_all_no_other_devices">Only do this if you have no other device you can verify this device with.</string>
+    <string name="secure_backup_reset_all_no_other_devices">Resetting your verification keys cannot be undone. After resetting, you won't have access to old encrypted messages, and any friends who have previously verified you will see security warnings until you re-verify with them.</string>
```

Comment:
create a new string here? and mark this one as <to delete> eventually? Check the Contributing.md for strings.

I think we can keep a short version and a longer one. Use the short in the bottomsheet and the long in the reset fragment?

Reply:
Also `won't`needs an escape